sys/drv/bcm2835_emmc.c

```diff
@@ -389,26 +419,36 @@ static int bcmemmc_set_prop(device_t *cdev, uint32_t id, uint64_t var) {
     case EMMC_PROP_RW_RCA:
       state->rca = var;
       break;
+    case EMMC_PROP_RW_ERRORS:
+      /* The only way to reset nternal error is to reset the entire controller
+       * using the `reset` method. */
+      state->errors = var & ~EMMC_ERROR_INTERNAL;
```

Comment:
Shouldn't be `state->errors = var | (state->errors & EMMC_ERROR_INTERNAL)`?

Reply:
No.
This is meant to be used to clear error flags. The exception is the `EMMC_ERROR_INTERNAL` flag, which can't be cleared by setting `ERRORS` property. The only way to clear it is to reset the controller.

Reply:
Sorry if it's obvious and I'm missing it. If I understand correctly, `var` is provided by the user, so e.g. if `var = 0` and `state->errors = EMMC_ERROR_INTERNAL`, then after the assignment `state->errors = 0`. Where is the problem in this reasoning?

Reply:
Ok, thanks for pointing that out, that was indeed a mistake on my side. I fixed it:
```c
state->errors = var | (state->errors & EMMC_ERROR_INTERNAL);
```